src/styles/components/_card.scss

```diff
@@ -0,0 +1,84 @@
+$card-border: 1px solid #e5e5e5;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px;
+  width: 200px;
+  border: $card-border;
+  border-radius: 5px;
+
+  &__img {
+    display: flex;
+    align-items: center;
+    width: 160px;
+    height: 134px;
+    margin: 32px 3px 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    line-height: 1.5;
+    margin-bottom: 4px;
+    color: $color-main-accent;
+  }
+
+  &__id {
+    font-size: 10px;
+    line-height: 1.4;
+    margin-bottom: 16px;
+    color: $color-secondary;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &-inscription {
+      font-size: 10px;
+      line-height: 1.4;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-inscription {
+      line-height: 1.5;
+      color: $color-secondary;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 1.13;
+      color: $color-main-accent;
+    }
+  }
+
+  &__submit {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    margin-bottom: 16px;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    width: 166px;
```

Comment:
flex element will take up all of the available space anyways
```suggestion
```

Reply:
Fixed ✔